filebeat/harvester/source.go

```diff
@@ -1,17 +1,13 @@
-package source
+package harvester
 
 import (
 	"io"
 	"os"
 )
 
-type LogSource interface {
+type Source interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Source should have comment or be unexported